Software Engineer Scala Connectivity.md

```diff
@@ -0,0 +1,54 @@
+[HolidayCheck](http://www.holidaycheck.de/) is deeply commited to providing the best experience in online travel for our customers. Over the past month we've been rewriting most of our backend using Scala and Finatra. Now we need help to bring our platform to the next level from a passionate
+
+# (Senior) Software Engineer – Scala / Connectivity
+
+## Your Tasks
+- Be part of our backend developer team driving our Scala/Akka-based travel search and booking platform
```

Comment:
What about writing 
"Drive backend development of our Scala/ Akka-based travel search and booking platform"
so that it matches the rest better?